<ui version="4.0" >
 <class>labelForm</class>
 <widget class="QWidget" name="labelForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>421</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>431</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="5" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Volume Name:</string>
       </property>
       <property name="buddy" >
        <cstring>volumeName</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="poolCombo" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Storage:</string>
       </property>
       <property name="buddy" >
        <cstring>storageCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="volumeName" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QSpinBox" name="slotSpin" >
       <property name="maximum" >
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Slot:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item rowspan="6" row="0" column="2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Execute Automount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="automountOnButton" >
         <property name="text" >
          <string>On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="automountOffButton" >
         <property name="text" >
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="storageCombo" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Pool:</string>
       </property>
       <property name="buddy" >
        <cstring>poolCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>71</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Label a Volume&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>81</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>131</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
